### PR TITLE
add docker development regtest enviroment

### PR DESCRIPTION
**Pull request by dni**
full docker regtest setup with all services. could potentially used for automated testing.

* lnbits
* bitcoin-core
* lnd
* clightning
* electrs
* mempool
* boltz

### CODE REVIEW

**Comment by dni** [comment]
i can, you planning to leave gh?

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      HOST: lnbits
+      LNBITS_BACKEND_WALLET_CLASS: "LndRestWallet"
+      LNBITS_DATA_FOLDER: "./data"
+      LND_REST_ENDPOINT: "https://lnd-2:8081/"
+      LND_REST_CERT: "./lnd/tls.cert"
+      LND_REST_MACAROON: "./lnd/data/chain/bitcoin/regtest/admin.macaroon"
+    ports:
+      - 5000:5000
+    volumes:
+      - ../lnbits:/app/lnbits
```
[the remark this answers is not in the document]
Comment (arcbtc) [reply docker/docker-compose.yml:19]: We're not leaving gh. Can you build from the legend repo instead?

**Comment by arcbtc** [comment]
Gave you commit access here, please add the files from this pr https://github.com/lnbits/legend-regtest-enviroment

**Comment by dni** [comment]
done. https://github.com/lnbits/legend-regtest-enviroment/commit/b732fe4d43a3fa4c8b3e0cb021faad82d7e3dead

**Pull request closed without merging**